README.md

```diff
@@ -0,0 +1,2 @@
+# Build
+![VS2019](https://github.com/T-rvw/OpenGL-Practice/actions/workflows/msbuild.yml/badge.svg?branch=main)
```

Comment:
You can modify it later by using `Muruc` to replace my username to show your main branch status.